notification/app/notification/services/guardian/ReportTopicRegistrationCounter.scala

```diff
@@ -8,16 +8,18 @@ import scala.concurrent.duration.DurationLong
 
 class ReportTopicRegistrationCounter(
   ws: WSClient,
-  registrationCounterUrl: String
+  reportUrl: String,
+  apiKey: String
 )(implicit ec: ExecutionContext) extends TopicRegistrationCounter {
 
   val lruCache: LruCache[PlatformCount] = new LruCache[PlatformCount](200, 1000, 3.days)
 
   override def count(topics: List[Topic]): Future[PlatformCount] = {
     lruCache(topics.toSet) {
       val topicParameters = topics.map(topic => "topics" -> topic.toString)
-      ws.url(registrationCounterUrl)
-        .withQueryStringParameters(topicParameters: _*)
+      val allParams = topicParameters ++ List("api-key" -> apiKey)
```

Comment:
can you not append a single tuple: `val allParams = topicParameters :+ ("api-key" -> apiKey)`